server/fleet/statistics.go

```diff
@@ -19,6 +19,10 @@ type StatisticsPayload struct {
 	VulnDetectionEnabled      bool   `json:"vulnDetectionEnabled"`
 	SystemUsersEnabled        bool   `json:"systemUsersEnabled"`
 	HostsStatusWebHookEnabled bool   `json:"hostsStatusWebHookEnabled"`
+	MDMMacOsEnabled           bool   `json:"MDMMacOsEnabled"`
```

Comment:
To follow camelCase convention in all other fields, these should be: `mdmMacOSEnabled` and `mdmWindowsEnabled `.

Reply:
I changed the JSON strings.
However the parameter has to start with capital otherwise Go lang will not export it.
So have to stay MDM